gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/ProcedureRequestMapperTest.java

```diff
@@ -66,8 +64,8 @@ public class ProcedureRequestMapperTest {
     private static final String STATUS_SEEN = "Status: Seen";
     private static final List<Encounter> ENCOUNTERS = getEncounterList();
     private static final Patient SUBJECT = createPatient();
-    private static final Meta META = MetaFactory.getMetaFor(META_WITH_SECURITY, META_PROFILE);
-    private static final Meta ALTERNATIVE_META = MetaFactory.getMetaFor(META_WITHOUT_SECURITY, META_PROFILE);
+    private static final Meta META = MetaFactoryUtil.getMetaFor(META_WITH_SECURITY, META_PROFILE);
+    private static final Meta ALTERNATIVE_META = MetaFactoryUtil.getMetaFor(META_WITHOUT_SECURITY, META_PROFILE);
```

Comment:
Maybe this also can be:

```suggestion
    private static final Meta META_WITHOUT_SECURITY_ADDED = MetaFactoryUtil.getMetaFor(META_WITHOUT_SECURITY, META_PROFILE);
```